instrumentation/nocode/src/main/java/com/splunk/opentelemetry/instrumentation/nocode/NocodeMethodInvocation.java

```diff
@@ -46,7 +46,7 @@ public Object getThiz() {
   /**
    * Please be careful with this, it's directly tied to @Advice.AllArguments.
    *
-   * @return
+   * @return @Advice.AllArguments - please be careful
```

Comment:
just curious what concerns did you have to add this warning?

Reply:
Jason pointed out that this is a mutable array and users of it should take care to know that it is tied to the given advice instrumentation.  https://github.com/signalfx/splunk-otel-java/pull/2184#discussion_r1955285508

Reply:
`@Advice.AllArguments` takes a parameter `readOnly` that defaults to `true`. This parameter controls whether method arguments can be mutated using the returned array. Reading https://github.com/signalfx/splunk-otel-java/pull/2184#discussion_r1955285508 I think that Jason was just concerned that array is a mutable type. I wouldn't be too concerned with this.